### PR TITLE
Recreate blog post PR 1142 (we did not have access rights)

### PR DESCRIPTION
**Pull request by alicejw-aws**
Tried to pull down team member's PR, [1142](https://github.com/opensearch-project/project-website/pull/1142) but access rights were denied. 

New PR created by copying file contents and images.

Signed-off-by: Alice Williams <alicejw@amazon.com>

### Check List
- [ x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by alicejw-aws** [comment]
Hi @krisfreedain, here is the blog post we had to recreate it, as we didn't have access rights to update it directly. Could you please approve, it's got your comments updated from the original PR. 
https://github.com/opensearch-project/project-website/pull/1142
thanks

File: _authors/hdhalter.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Heather Halter
+short_name: hdhalter
+photo: '/assets/media/authors/hdhalter.jpg'
```
Comment (krisfreedain) [inline _authors/hdhalter.markdown:4]: looks like the file uploaded is .jpeg but this references .jpg

File: _posts/2022-11-08-thudtest.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+When I first started my career as a technical writer about 20 years ago, success was measured by the volume of content produced. Back then, everything was printed and bound. When you dropped a stack of paper on a desk or the ground, it would make an audible *thud*. The theory went that the louder the thud, the more content you had produced, and the happier your customers were.
+
+Fast-forward about 10 years, and it was all about simplification: Only provide the minimum amount of content that your customers need to accomplish their tasks. So we set about removing information and only documenting what was absolutely necessary.
+
+In the years that followed, technical writers continued to face tough choices. We know that we have to write for customers first. We know that writing simply is hard. We know that we need to insist on the highest standards. We know that words are expensive to translate and resource intensive to change as the user interface changes.
+
+And we know that product managers still want to use the thud test to measure documentation output. <p style="max-width: 50%; float: right;">
+![Stack of papers](../assets/media/blog-images/stack%20of%20papers.jpg)</p>
```
Comment (krisfreedain) [inline _posts/2022-11-08-thudtest.md:20]: looks like the file uploaded is .jpeg but this references .jpg

also, the path to your images needs to be resolved. See this example from another blog post. 

Example from another post: 
Jenkins Pipeliens Flowchart{: .img-fluid }

you'll need to resolve all the image locations, and redo your path within each image line through this blog post.

**Review by krisfreedain — CHANGES_REQUESTED** [review]
multiple issues with images - please resolve and we'll push this out - thank you

File: _authors/hdhalter.markdown (hunk 1 above)
Comment (alicejw-aws) [reply _authors/hdhalter.markdown:4]: so sorry. fixing now!

File: _posts/2022-11-08-thudtest.md (hunk 2 above)
Comment (alicejw-aws) [reply _posts/2022-11-08-thudtest.md:20]: ok thanks!

**Comment by alicejw-aws** [comment]
hi @krisfreedain , i've updated the image paths and moved to similar directory with name of blog like the example you shared. please let me know if this looks correct now. 
thanks so much for your help!

**Review by krisfreedain — APPROVED** [review]
that last edit fixed it @alicejw-aws - thanks

**Pull request merged**